packages/@webex/plugin-presence/test/integration/spec/presence.ts

```diff
@@ -0,0 +1,211 @@
+/*!
+ * Copyright (c) 2015-2020 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import '@webex/internal-plugin-mercury';
+import '@webex/plugin-presence';
+
+import {assert} from '@webex/test-helper-chai';
+import WebexCore from '@webex/webex-core';
+import testUsers from '@webex/test-helper-test-users';
+import {expectEvent} from '@webex/test-helper-mocha';
+import sinon from 'sinon';
+
+// SPARK-413317
+describe.skip('plugin-presence', function () {
```

Comment:
this should probably be fixed at some point

Reply:
Yes, this should be fixed but not in the scope of this PR.